### ISSUE TITLE
Use Connectmed API for gathering service provider information

### ISSUE THREAD

**Issue by rennanharo** [Maronato/mck-medicos#1]
**TL;DR**: GAMA (Connectmed) uses a public API on [their website](https://gamasaude.com.br/rede-credenciada/busca-de-rede) for querying service provider information. We can try to leverage this API in order to keep Gama Médicos always updated and not relying on an Excel Spreadsheet as the data source for the application.

I found out about this API by monitoring the network calls on their website once you search for a service provider.
 
As this API has no documentation at all, I did a quick investigation to try and understand what are the endpoints and their respective route/query params.

To this moment, I could find two endpoints that could be of use - bear in mind that **I'm guessing** the route params based on some initial analysis: 
1. **_statusPlano_**: returns information about a particular plan.
**Endpoint**: `https://wwwt.connectmed.com.br/saudeweb/redeatendimento/api/4/statusPlano/`
**Route params:** plan code
**Example:** `https://wwwt.connectmed.com.br/saudeweb/redeatendimento/api/4/statusPlano/233` returns information about the plan with code 233 - which happens to be "MASTER - OURO"

2. **_prestadoresBairro_**: returns information about service providers given a particular query
**Endpoint:** `https://wwwt.connectmed.com.br/saudeweb/redeatendimento/api/4/prestadoresBairro/`
**Route params**: `plan code`/`especialidade`/`estado`/`cidade`/`bairros`/`tipo de serviço`
**Example**: `https://wwwt.connectmed.com.br/saudeweb/redeatendimento/api/4/prestadoresBairro/233/HOSPITAL/SP/SAO%20PAULO/Todos/8` returns information about hospital service providers in São Paulo, SP, in all neighborhoods and that are covered by the plan 233 (MASTER - OURO).
<br>As per the source code on website, the parameter `tipo de serviço` can be mapped to the following codes:
```
1 - Consultório médico/clínica
8 - Hospital
2 - Maternidade
4 - Pronto-socorro
6 - Serviço de diagnóstico
7 - Serviço de terapia 
```
